android/src/main/java/com/mattermost/networkclient/Extensions.kt

```diff
@@ -0,0 +1,66 @@
+package com.mattermost.networkclient
+
```

Comment:
Doh good idea, didn't even think of holding the extensions in their own file - makes a lot more sense 👍